docker/Makefile

```diff
@@ -62,7 +62,8 @@ dcmqi.generate_package: prereq.pull_dockcross
 
 # Build "dcmqi" docker image
 dcmqi.docker_image:
-	tar -xzvf $(ROOT_DIR)/$(BUILD_DIR)/dcmqi-build/dcmqi-linux.tar.gz -C $(DOCKER_DIR)/dcmqi
+	PACKAGE_FILE_NAME = $(shell ls /home/ubuntu/dcmqi/build/dcmqi-build/_CPack_Packages/linux/TGZ/dcmqi-*-linux*.tar.gz)
+	tar -xzvf $(PACKAGE_FILE_NAME) -C $(DOCKER_DIR)/dcmqi
```

Comment:
Yes, I know, but I am not sure it would be faster than CircleCI. I didn't time it, but it takes a while to build even on a powerful platform. I should time it to get a better idea!